core/domain/classifier_domain.py

```diff
@@ -277,6 +277,27 @@ def status(self):
     def training_data(self):
         return self._training_data
 
+    def update_state_name(self, state_name):
```

Comment:
I'm confused about this. If the job is scoped to a particular exploration version, then the state name is not changeable. So why have this?